app/helpers/users_helper.rb

```diff
@@ -6,4 +6,17 @@ def user_role_options
       ['Admin', 'admin']
     ]
   end
+
+  def th_autosortable(column_name, type, local_assigns)
+    # Throw a flag unless sorting by string, int, or float
+    raise 'Bad datatype' unless %w(string string-ins int float).include? type
+    content_tag :th, data: { sort: type } do
+      safe_join [column_name, autosort_arrow_span(local_assigns)], ' '
+    end
+  end
+
+  def autosort_arrow_span(local_assigns)
+    return '' unless local_assigns[:autosortable]
+    content_tag(:span, '[-]', class: 'arrow')
+  end
```

Comment:
The goods news is you're not getting old, but we need to move this code out to a shared helper. I was thinking ```tables_helper.rb``` that contains all table logic. Thoughts?